code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -2215,3 +2331,21 @@
 					sleep(1 SECOND) //let us do that combo shit people like with throwing
 					src.force_laydown_standup()
 
+/// Looks for the kind_of_target atoms within range, and throws the user an input
+/// Valid kinds: "mob", "obj", "both"
+/mob/living/proc/get_targets(range = 1, kind_of_target = "mob")
+	if(!isturf(get_turf(src))) return
+
+	. = list()
+
+	if(kind_of_target == "mob" || kind_of_target == "both")
+		for(var/mob/M in view(range, get_turf(src)))
+			if(M == src)
+				continue
+			. |= M
+
+	if(kind_of_target == "obj" || kind_of_target == "both")
+		for(var/obj/O in view(range, get_turf(src)))
```

Comment:
could shove for (for atom movable in view) into a varlist and then depending on target get just the mobs or objects from the list

Reply:
I think I did that, though I'm not really sure what you mean by that.